templates/zerver/integrations/index.html

```diff
@@ -25,7 +25,7 @@
                 <div id="integration-main-text">
                     <header>
                         <h1 class="portico-page-heading">
-                            {% trans %}Over 60 native integrations.{% endtrans %}
+                            {% trans %}Over 90 native integrations.{% endtrans %}
```

Comment:
Maybe we should move the number into a variable?  Otherwise translators will have to re-translate every time we change the number...